fooof/tests/test_synth.py

```diff
@@ -7,6 +8,27 @@
 ###################################################################################################
 ###################################################################################################
 
+def test_param_iter():
+
```

Comment:
Add a simple 2 value (reflect AP) and 3 value (reflects osc) test case. 

Add comment labels:
`# Tests 2 value (used for AP generation)`

`# Test 3 value (used for single osc generation)`

`# Test n value (test generalization)`

`# Test multiple oscillations case`